lua/InternalEvents.h

```diff
@@ -213,5 +188,4 @@ struct BeamHit
         NEW_ROOM   // treat beam hit as new room from last frame
     };
 };
-
-#endif // INTERNALEVENTS_H
+#endif // INTERNALEVENTS_H
```

Comment:
It needs a linebreak here too.
```suggestion
#endif // INTERNALEVENTS_H

```